pkg/handlers/utils.go

```diff
@@ -1,9 +1,17 @@
 package handlers
 
 import (
+	"strings"
+
 	"github.com/coinbase/rosetta-sdk-go/types"
 )
 
+const (
+	proxyContractPrefix          = "0x42"
```

Comment:
all the address with "0x42", do we wan to add more 0000 to avoid potential personal accs?

Reply:
Actually I think I can use `proxyContractFilter` (the longer one)

Reply:
This is only for short circuit evaluation, so longer one is good 👍

Reply:
Done